File: 01-data-preprocessing/missing_data.r
df <- read.csv("movies.csv")
print(head(df))

if (any(is.na(df$RunTime))) {
  cat("Column with missing values\n")
  print(head(df$RunTime))
  df$RunTime[is.na(df$RunTime)] <- mean(df$RunTime, na.rm = TRUE)
  cat("Missing values filled with the mean\n")
  print(head(df$RunTime))
} else {
  cat("No missing values in the column\n")
}

File: 01-data-preprocessing/missing_data.py
import pandas as pd

df = pd.read_csv("movies.csv")
print(df.head())

if df["RunTime"].isnull().values.any():
    print("Column with missing values")
    print(df["RunTime"].head())
    df["RunTime"] = df["RunTime"].fillna(df["RunTime"].mean())
    print("\nMissing values filled with mean")
    print(df["RunTime"].head())
else:
    print("No missing values in column")
